docs/changelogs/v0.31.0.md

```diff
@@ -1,39 +1,128 @@
-This is the third release candidate for the upcoming Cozystack v0.31.0 release.
-The release notes show changes accumulated since the release of previous version, Cozystack v0.30.0.
+Cozystack v0.31.0 is a significant release that brings new features, key fixes, and updates to underlying components.
+This version enhances GPU support, expands automation (like autoscaling and CI/CD), and introduces a more robust release process for greater stability .
+Below we outline the highlights in each area for current users and developers.
 
-Cozystack 0.31.0 further advances GPU support, monitoring, and all-around convenience features.
+## Major Features and Improvements
 
-## New Features and Changes
+### GPU support for tenant Kubernetes clusters
 
+Cozystack now integrates NVIDIA GPU Operator support for tenant Kubernetes clusters.
+This allows platform users to run GPU-enabled applications such as AI/ML in their own clusters.
```

Comment:
```suggestion
This allows platform users to run GPU-powered applications for AI/ML in their own clusters.
```

Reply:
AI/ML are not the only applications using GPU, I guess.